Example/Pods/TextAnnotation/TextAnnotation/Classes/CustomSwitch.swift

```diff
@@ -0,0 +1,278 @@
+import Foundation
+
+public class CustomSwitch: NSControl {
+  public static let bundle = Bundle(for: CustomSwitch.self)
+  
+  public var tintColor = NSColor(calibratedRed: 0.243, green: 0.235, blue: 0.243, alpha: 1.0) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var knobBackgroundColor = NSColor(calibratedWhite: 1.0, alpha: 1.0) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var disabledBorderColor = NSColor(calibratedWhite: 0.0, alpha: 0.2) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var disabledBackgroundColor = NSColor(calibratedRed: 0.243, green: 0.235, blue: 0.243, alpha: 1.0)  {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var inactiveBackgroundColor = NSColor(calibratedWhite: 0.0, alpha: 0.3) {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var animationDuration: TimeInterval = 0.4
+  public var inactiveIcon = bundle.image(forResource: "T_shadow_selected") {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var activeIcon = bundle.image(forResource: "T_shadow_unselected") {
+    didSet {
+      reloadLayer()
+    }
+  }
+  public var secondIcon = bundle.image(forResource:"T_outline") {
+    didSet {
+      reloadLayer()
+    }
+  }
+  
+  override public var isEnabled: Bool {
+    didSet {
+      reloadLayerAnimated(animated: true)
+    }
+  }
+  
+  lazy var kBorderLineWidth: CGFloat = 0
+  let kEnabledOpacity: Float = 1.0
+  let kDisabledOpacity: Float = 0.5
+  var dragEvents = 0
+  
+  public var isOn: Bool = false
+  public var isActive: Bool = false
+  public var hasDragged: Bool  = false
+  public var isDraggingTowardsOn: Bool = false
+  public var lockInteraction: Bool = false
+  public let rootLayer = CALayer()
+  public let backgroundLayer = CALayer()
+  public let knobLayer = CALayer()
+  public let icon1Layer = CALayer()
+  public let icon2Layer = CALayer()
+  
+  override public var acceptsFirstResponder: Bool {
+    get { true }
+  }
```

Comment:
@dworak lets please avoid using this explicit `gets` and `return` conditions.
No need for them in Swift 5.1.

Reply:
@dworak it would be great if you could finally check my SwiftLint PR and we could integrate it into the project. Thank you -> https://github.com/blackbeltlabs/zappy/pull/78